apps/mark-scan/frontend/src/app.tsx

```diff
@@ -81,8 +84,21 @@ export function App({
                 api={uiStringsApi}
                 disabled={!enableStringTranslation}
               >
-                <AppRoot hardware={hardware} reload={reload} logger={logger} />
-                <SessionTimeLimitTracker />
+                <ThemeConsumer>
+                  {(currentTheme) => (
+                    <React.Fragment>
+                      {currentTheme.isVisualModeDisabled && (
```

Comment:
Assuming this isn't behaviour specific to mark-scan alone, does it make sense to move this logic to `<AppBase>`?

Reply:
From my read of the product brief, we'd also need audio-only mode in VxScan. @mattroe can you confirm before we make this change?

Reply:
FWIW, moving it to a common component like `<AppBase>` shouldn't affect apps that never set `isVisualModeDisabled` to `true`. The potentially confusing piece here is that it's a global setting at the shared theme level, but the corresponding behaviour is only being implemented in one app -- whereas, the toggle to control it is exposed in the shared `<DisplaySettings>` component, which is rendered currently in Mark, Scan, and Mark-Scan. (We might need to make the sensory toggle settings tab optionally enabled, for that matter.)

Reply:
Ah yeah I just wanted to clarify since I realized I don't know if we can make this assumption
> Assuming this isn't behaviour specific to mark-scan alone

Agreed on moving to `<AppBase>` in that case, and also hiding the toggle from apps (if any) that don't need audio-only mode.

Reply:
[b03bf3d](https://github.com/votingworks/vxsuite/pull/4518/commits/b03bf3d734b28c9f0e1c5d325db897dd85d7e0a1)

Reply:
@kshen0 We don't have an audio-tactile interface on VxScan so it's not applicable. It's overall a better experience to just cast an audio-tactile interface ballot on VSAP than to try to cast a ballot you may not be able to verify into a piece of precinct scanning hardware that you may not be able to see.

Reply:
https://github.com/votingworks/vxsuite/pull/4518/commits/84bda2dff6d86ee28252588391a46d6efe5de0da updates all naming to variations of "Audio/Video Only", `audioVideoOnlySettings` etc.

Reply:
@kofi-q heads up I reverted back to rendering `VisualModeDisabledOverlay` from `mark-scan/app.tsx` because

1. It's used in mark-scan only
2. Rendering from `app_base` means it takes priority over the `ErrorBoundary` in app-specific `app.tsx` files. So an error would be rendered under the overlay, but we probably want the error to be rendered over the overlay.

Reply:
^ ah, that makes sense -- SGTM!